chemprop/args.py

```diff
@@ -84,6 +84,11 @@ class CommonArgs(Tap):
     """Number of workers for the parallel data loading (0 means sequential)."""
     batch_size: int = 50
     """Batch size."""
+    atom_descriptors: Literal['None', 'feature', 'descriptor'] = 'None'
+    """Custom extra atom descriptors. feature: used as atom features to featurize a given molecule. 
+    descriptor: used as descriptor and concatenated to the machine learned atomic representation"""
+    atom_descriptors_path: str = None
+    """Path to the extra atom descriptors"""
```

Comment:
Adding a period for consistency.

```suggestion
    """Path to the extra atom descriptors."""
```